Add quarto-live official backend use of Pyodide to related projects (#4997)
pyodide/pyodide: docs/project/related-projects.md

  [Quarto](https://quarto.org/) document formats like
  HTML Documents, RevealJS, Books, and Websites.
- [PyCafe](https://py.cafe) lets you host, edit, and share Python apps in your browser with a single click.
- [quarto-live](https://github.com/r-wasm/quarto-live) uses Pyodide

>   to create interactive code cells and exercises within a variety of
>   [Quarto](https://quarto.org/) document.

Reviewer: ```suggestion
  to create interactive Python code cells and exercises in [Quarto](https://quarto.org/) documents.
```
